diff --git a/node_modules/@commitlint/config-conventional/index.js b/node_modules/@commitlint/config-conventional/index.js
index a2e917d..97981be 100644
--- a/node_modules/@commitlint/config-conventional/index.js
+++ b/node_modules/@commitlint/config-conventional/index.js
@@ -30,6 +30,12 @@ module.exports = {
 				'revert',
 				'style',
 				'test',
+				'xodofeature',
+                'xodoactionfeature',
+                'support',
+                'xodofix',
+                'dev',
+                'analytics'
 			],
 		],
 	},
